src/main/java/org/churchbooks/churchbooks/transactions/TransactionLoader.java

```diff
@@ -0,0 +1,70 @@
+package org.churchbooks.churchbooks.transactions;
+
+import com.webcohesion.ofx4j.domain.data.MessageSetType;
+import com.webcohesion.ofx4j.domain.data.ResponseEnvelope;
+import com.webcohesion.ofx4j.domain.data.banking.BankingResponseMessageSet;
+import com.webcohesion.ofx4j.domain.data.common.Transaction;
+import com.webcohesion.ofx4j.io.AggregateUnmarshaller;
+import com.webcohesion.ofx4j.io.OFXParseException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.net.URLConnection;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TransactionLoader {
```

Comment:
Do you think this should be a Service so it can be injected via DI?

Reply:
Intended TransactionLoader to be a parsing utility, can mark with @Component instead to still allow DI?